src/widget/wsearchlineedit.cpp

```diff
@@ -88,6 +88,7 @@ WSearchLineEdit::WSearchLineEdit(QWidget* pParent)
     setIconSize(QSize(0, 0));
     setInsertPolicy(QComboBox::InsertAtTop);
     setMinimumSize(0, 0);
+    setDuplicatesEnabled(false);
```

Comment:
isn't this default anyway?

Reply:
Yes, that's the default. Setting it should be redundant and not solve any issues.

https://doc.qt.io/qt-5/qcombobox.html#duplicatesEnabled-prop

I experience both duplicates and disappearing entries (currently using `main`). Unfortunately I haven't yet discovered a pattern when and how this is happening.

Reply:
I was not able to create duplicates for a long time, so I thought I caught all cases that cause inserts.

Reply:
Does it mean we can remove the es line before merge? At least the duplicate issue is unrelated to this PR

Reply:
removed